src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -0,0 +1,380 @@
+					overflow -= this.getWidth(breadcrumbs[currentIndex].elm)
+					hiddenCrumbs.push(breadcrumbs[currentIndex])
+					i++
+				}
+				// We only update the hidden crumbs if they have changed,
+				// otherwise we will run into an infinite update loop.
+				if (!this.arraysEqual(this.hiddenCrumbs, hiddenCrumbs.sort())) {
+					this.hiddenCrumbs = hiddenCrumbs
+				}
+			}
+		},
+		arraysEqual(a, b) {
+			if (a.length !== b.length) return false
+			if (a === b) return true
+			if (a === null || b === null) return false
+
+			for (let i = 0; i < a.length; ++i) {
+				if (a[i] !== b[i]) return false
+			}
+			return true
+		},
+		getTotalWidth(breadcrumbs) {
+			return breadcrumbs.reduce((width, crumb, index) => width + this.getWidth(crumb.elm), 0)
+		},
+		getWidth(el) {
+			const hide = el.classList.contains('hidden')
+			el.classList.remove('hidden')
+			const w = el.offsetWidth
+			if (hide) {
+				el.classList.add('hidden')
+			}
+			return w
+		},
+		cancelEvent(e) {
+			e.stopPropagation()
+			e.preventDefault()
+			return false
+		},
+		dragstart(e) {
+			return this.cancelEvent(e)
+		},
+		dropped(path, e) {
+			this.cancelEvent(e)
+			this.$emit('dropped', path)
+			this.$refs.actions.opened = false
+			const crumbs = document.querySelectorAll('.crumb')
+			crumbs.forEach((f) => { f.classList.remove('over') })
+			return false
+		},
+		dragOver(e) {
+			if (e.preventDefault) {
+				e.preventDefault()
+			}
+			return false
+		},
+		dragEnter(e) {
+			// Get the correct element, in case we hover a child.
+			if (e.target.closest) {
+				const target = e.target.closest('.crumb')
+				if (target.classList && target.classList.contains('crumb')) {
+					const crumbs = document.querySelectorAll('.crumb')
+					crumbs.forEach((f) => { f.classList.remove('over') })
+					target.classList.add('over')
+				}
+			}
+		},
+		dragLeave(e) {
+			// Don't do anything if we leave towards a child element.
+			if (e.target.contains(e.relatedTarget)) {
+				return
+			}
+			// Get the correct element, in case we leave directly from a child.
+			if (e.target.closest) {
+				const target = e.target.closest('.crumb')
+				if (target.contains(e.relatedTarget)) {
+					return
+				}
+				if (target.classList && target.classList.contains('crumb')) {
+					target.classList.remove('over')
+				}
+			}
+		},
+		/**
+		 * Check for each crumb if we have to hide it and
+		 * add it to the array of all crumbs.
+		 *
+		 * @param {Array} crumbs The array of all crumbs
+		 * @param {Array} newCrumbs The array of the crumbs to hide and add
+		 */
+		addCrumbs(crumbs, newCrumbs) {
+			newCrumbs.forEach((crumb, i) => {
+				if (crumb.elm && crumb.elm.classList) {
+					if (this.hiddenCrumbs.includes(crumb)) {
+						crumb.elm.classList.add('hidden')
+					} else {
+						crumb.elm.classList.remove('hidden')
+					}
+				}
+				crumbs.push(crumb)
+			})
+		},
+	},
+	render: function(createElement) {
```

Comment:
I would love a little bit more details on this function. If someone have to maintain it afterwards. Extra comments will not hurt the bundle size :wink:

Reply:
Will do that. I currently also check how often it triggers.

Reply:
As stated above, I removed the `mounted` hook.